### PR TITLE
Fix issue with disappearing content

### PR DESCRIPTION
**Pull request by fjvallarino**
@santiagonoguez @stulees this fixes the problem we were having when accessing a presentation after opening the Financial component. 

A minor side comment: should we rename `factory.selected` to `factory.selectedComponent`? It's a bit confusing (I'm to blame, since I chose that name)

@Rise-Vision/apps FYI

### CODE REVIEW

File: web/scripts/template-editor/components/directives/dtv-component-financial.js [hunk 1]
```diff
@@ -18,13 +18,13 @@ angular.module('risevision.template-editor.directives')
             $scope.exitingSymbolSelector = false;
 
             // TODO: hardcoding category for now until templates have component surface category attribute
-            $scope.category = "currencies";
+            $scope.category = 'currencies';
```
Comment (santiagonoguez) [inline web/scripts/template-editor/components/directives/dtv-component-financial.js]: @fjvallarino are these quote changes needed ? @stulees is now working in this same file and these style changes may cause conflicts when merging.

Comment (fjvallarino) [reply web/scripts/template-editor/components/directives/dtv-component-financial.js]: @santiagonoguez not really, it's just that the linter complains about them. We should stick to a single type of quotes (in Apps it's single quotes). I can revert this commit and fix in a later PR

Comment (santiagonoguez) [reply web/scripts/template-editor/components/directives/dtv-component-financial.js]: It's the same for me, nice to know about quotes in Apps !

@stulees what do you prefer for this PR ?

Comment (stulees) [reply web/scripts/template-editor/components/directives/dtv-component-financial.js]: @fjvallarino @santiagonoguez I don't mind. I'll make the same quote changes for this file in my eventual PR and that way neither of us will run into a conflict.

**Comment by santiagonoguez** [comment]
@fjvallarino are you merging this one ?

**Comment by fjvallarino** [comment]
@santiagonoguez I forgot about it! I'll merge now

**Pull request merged**